Lib/tracemalloc.py

```diff
@@ -171,16 +171,16 @@ def __repr__(self):
 @total_ordering
 class Traceback(Sequence):
     """
-    Sequence of Frame instances sorted from the most recent frame
-    to the oldest frame.
+    Sequence of Frame instances sorted from the oldest frame
+    to the most recent frame.
     """
     __slots__ = ("_frames",)
 
     def __init__(self, frames):
         Sequence.__init__(self)
         # frames is a tuple of frame tuples: see Frame constructor for the
         # format of a frame tuple
-        self._frames = frames
+        self._frames = tuple(reversed(frames))
```

Comment:
Please extend the comment to explain that the C module gives frames from the most recent to the oldest, but that ther reverse order is expected in the Python API.